src/pages/member/ViewMember.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+function ViewBookCatalog() {
+  const [members, setData] = useState({});
+  const { id } = useParams();
+
+  useEffect(() => {
+    const token = localStorage.getItem("token");
+
+    fetch(`http://localhost:3000/api/members/${id}`, {
+      method: "GET",
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    })
+      .then((response) => {
+        if (!response.ok) throw new Error("Network response was not ok");
+        return response.json();
+      })
+      .then((data) => {
+        console.log("Data", data);
+        setData(data);
+      })
+      .catch((error) => {
+        console.error("Error", error);
+      });
+  }, [id]);
+
+  return (
+    <div className="container mx-auto p-4">
+      <h1 className="text-2xl font-bold mb-4">Member Information</h1>
+
+      <div className="flex mt-4 py-4">
+        <button className="bg-gray-500 text-white py-2 px-5 rounded-lg mr-4">
+          Back
```

Comment:
Please create Button.jsx component for any type button and then just past the props